client/asset/dcr/dcr.go

```diff
@@ -461,7 +460,7 @@ func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network)
 			return nil, err
 		}
 	} else if makeCustomWallet, ok := customWalletConstructors[cfg.Type]; ok {
-		dcr.wallet, err = makeCustomWallet(cfg, chainParams, logger)
+		dcr.wallet, err = makeCustomWallet(cfg.Settings, chainParams, logger)
```

Comment:
What's the motivation for switching from `cfg` to `cfg.Settings`? This could affect other external implementations, although I haven't checked. A receiver of `cfg` can access `cfg.Settings` if that's all it cares about.

Reply:
Please take a look, but I think that based on the structure of `asset.WalletConfig`, it makes sense to restrain any `Wallet`-implementation-specific configuration to the `Settings` `map`.

Reply:
The other implementation I'm aware of only accesses the `Settings` map.